packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -437,8 +437,8 @@ module.exports = {
           '!**/*.json',
           '!**/__tests__/**',
           '!**/?(*.)(spec|test).*',
-          '!src/setupProxy.js',
-          '!src/setupTests.*',
+          '!**/src/setupProxy.*',
```

Comment:
Here's what i was referring to @Timer. It would still complain when running `yarn start` from root of CRA, when the rule was `!src/setupProxy.*`